de.metas.business/src/main/java/de/metas/customs/CustomsInvoiceRepository.java

```diff
@@ -145,6 +150,35 @@ I_C_Customs_Invoice save(@NonNull final CustomsInvoice customsInvoice)
 		return customsInvoiceRecord;
 	}
 
+	public List<CustomsInvoiceLine> retrieveLines(@NonNull final CustomsInvoiceId customsInvoiceId)
```

Comment:
why not get the customs invoice (entity root) here?
@teosarca wdyt?

Reply:
fixed!